submissions/all_submissions.php

```diff
@@ -14,10 +14,14 @@
 
 $submissions = getAllSubmissions();
 
+// we should try and group by coursework_id e.g. we first get a list of all submissions for Agile Development,
+// then a list of all submissions for Java Programming, then Business Systems, etc. see PDO::FETCH_GROUP
```

Comment:
Probably unnecessary to have this in the document.
But if needed should use Multi Line PHP comments.

```suggestion
/*we should try and group by coursework_id e.g. we first get a list of all submissions for Agile Development,
 then a list of all submissions for Java Programming, then Business Systems, etc. see PDO::FETCH_GROUP */
```

Reply:
It was more to remind us in sprint 2 that it can be done better, but I think you're right that it's unecessary to have it as a comment in the file. We could remove that comment entirely and I could add a note for myself in Teams?

Reply:
Yes if not needed make a User Card on the Product Backlog so it can be assigned to the 2nd sprint.
Thanks